vogon/settings.py

```diff
@@ -198,8 +198,8 @@
 
 CONCEPTPOWER_USERID = os.environ.get('CONCEPTPOWER_USERID')
 CONCEPTPOWER_PASSWORD = os.environ.get('CONCEPTPOWER_PASSWORD')
-CONCEPTPOWER_ENDPOINT = os.environ.get('CONCEPTPOWER_ENDPOINT')
-CONCEPTPOWER_NAMESPACE = os.environ.get('CONCEPTPOWER_NAMESPACE')
+CONCEPTPOWER_ENDPOINT = os.environ.get('CONCEPTPOWER_ENDPOINT', 'http://chps.asu.edu/conceptpower/rest/')
```

Comment:
and let's not put that url here. even though it is in the github commit history now, we want to keep bot traffic to our servers as low as possible